src/status_im2/contexts/syncing/scan_sync_code/animation.cljs

```diff
@@ -0,0 +1,61 @@
+(ns status-im2.contexts.syncing.scan-sync-code.animation
```

Comment:
Just moved the animation code from `view` to this namespace, since it was hard to read ther. I hope now it's clearer